api/src/main/java/edu/cornell/mannlib/vitro/webapp/controller/freemarker/IndividualListController.java

```diff
@@ -152,11 +155,17 @@ public static int getPageParameter(VitroRequest request) {
         return SearchQueryUtils.getPageParameter(request);
     }
 
-    public static IndividualListResults getResultsForVClass(String vclassURI, int page, String alpha, VitroRequest vreq)
+    public static IndividualListResults getResultsForVClass(String vclassURI,
+            int page, String alpha, VitroRequest vreq)
     throws SearchException{
    	 	try{
+            ConfigurationProperties props = ConfigurationProperties.getBean(vreq);
+            boolean languageFilter = Boolean.valueOf(props.getProperty(
+                    LANGUAGE_FILTER_PROPERTY, "false"));
             List<String> classUris = Collections.singletonList(vclassURI);
-			IndividualListQueryResults results = buildAndExecuteVClassQuery(classUris, alpha, page, INDIVIDUALS_PER_PAGE, vreq.getWebappDaoFactory().getIndividualDao());
+			IndividualListQueryResults results = buildAndExecuteVClassQuery(
+			        classUris, alpha, ((languageFilter) ? vreq.getLocale() : null),
```

Comment:
Can we move this code inside buildAndExecuteVClassQuery 
((languageFilter) ? vreq.getLocale() : null)
I mean you are repeating this code at two invocation, maybe you can beside method
buildAndExecuteVClassQuery(
			List<String> vclassURIs, String alpha, Locale locale, int page,
			int pageSize, IndividualDao indDao) {...}
also create
buildAndExecuteVClassQuery(
			List<String> vclassURIs, String alpha, boolean languageFilter, int page,
			int pageSize, IndividualDao indDao){
return buildAndExecuteVClassQuery(vclassURIs, alpha, ((languageFilter) ? vreq.getLocale() : null), page, pageSize, indDao);
}

Reply:
done